addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/__tests__/environment-sc-cidr-service.test.js

```diff
@@ -283,6 +331,264 @@ describe('EnvironmentScCidrService', () => {
       expect(service.authorizeSecurityGroupIngress).toHaveBeenCalledWith({}, authorizeParams);
     });
 
+    it('should not call anything since request has different protocol-port combinations', async () => {
```

Comment:
How does this not throw an exception/error. The updateRequest basically didn't work/go through. No?